app.js

```diff
@@ -3,29 +3,37 @@ var express = require('express'),
     compression = require('compression'),
     helmet = require('helmet'),
     cors = require('cors'),
-    rateLimit = require('express-rate-limit'),
     throng = require('throng'),
-    bodyParser = require('body-parser');
+    bodyParser = require('body-parser'),
+    Bitly = require('bitly'),
+    ExpressBrute = require('express-brute'),
+    RedisStore = require('express-brute-redis');
 
 Habitat.load();
 
 var app = express(),
     env = new Habitat(),
-    Bitly = require('bitly'),
     bitly = new Bitly(env.get('TOKEN')),
     port = env.get('PORT'),
+    redisUrl = env.get('REDIS_URL'),
     workers = env.get('WEB_CONCURRENCY') || 1,
     allowedOrigin = env.get('ALLOWED_ORIGIN');
 
+var bruteforce;
+
+if (redisUrl) {
+  bruteforce = new ExpressBrute(new RedisStore(require('redis-url').parse(redisUrl)));
+} else {
+  bruteforce = new ExpressBrute(new ExpressBrute.MemoryStore());
```

Comment:
Here, and above, we're going to need to use the second options param to specify `proxyDepth` and set it to 1 (production) or 0 (dev), so that it rate limits the X-Forwarded-For Header when behind the Heroku Router.